pysrc/scripts/sampling.py

```diff
@@ -19,12 +19,13 @@
 def sample_csv(file_path: str,
                sampling_method: str,
                sampling_rate: float,
-               config: GlobalConfiguration) -> list[list[tuple[str, str, float]]]:
-    """Sample a single file with a certain method and rate
-    and create a new tmp file. Returns the path to the sampled file.
+               config: GlobalConfiguration) -> list[tuple[str, str, float]]:
+    """Sample every single column of file seperately with a certain method and rate
+    and create a new tmp file for every column. Returns a list of tuples including
+    the path, rate, method of the  column of the sampled file.
```

Comment:
```suggestion
    the path, method, rate of the  column of the sampled file.
```

This should be the correct order.